docs/src/concepts.md

```diff
@@ -0,0 +1,160 @@
+
+The Element is composed of two main schemas, `scan` and `imaging`. To handle
+several use cases of this pipeline, we have designed two alternatives to `imaging` schemas,
+including `imaging_no_curation` and `imaging_preprocess`.
+
+
+- `imaging` module - Multiple scans are acquired during each session and each scan is
+processed independently.
+
+![element-calcium-imaging diagram](https://raw.githubusercontent.com/datajoint/element-calcium-imaging/main/images/attached_calcium_imaging_element.svg)
+
+- `imaging_no_curation` module - Same as `imaging` module, without the Curation table.
+
+![element-calcium-imaging diagram](https://raw.githubusercontent.com/datajoint/element-calcium-imaging/main/images/attached_calcium_imaging_element.svg)
+
+
+- `imaging_preprocess` module - Same as `imaging` module. Additionally, pre-processing
+steps can be performed on each scan prior to processing with Suite2p or CaImAn.
+![element-calcium-imaging diagram](https://raw.githubusercontent.com/datajoint/element-calcium-imaging/main/images/attached_calcium_imaging_element_preprocess.svg)
+
+
+### `lab` schema ([API docs](../api/workflow_calcium_imaging/pipeline/#workflow_calcium_imaging.pipeline.Equipment))
+
+| Table | Description |
+| --- | --- |
+| Equipment | Scanner metadata |
+
+### `subject` schema ([API docs](https://datajoint.com/docs/elements/element-animal/api/element_animal/subject))
+- Although not required, most choose to connect the `Session` table to a `Subject` table.
+
+| Table | Description |
+| --- | --- |
+| Subject | Basic information of the research subject |
+
+### `session` schema ([API docs](https://datajoint.com/docs/elements/element-session/api/element_session/session_with_datetime))
+
+| Table | Description |
+| --- | --- |
+| Session | Unique experimental session identifier |
+
+
+### `scan` schema ([API docs](https://datajoint.com/docs/elements/element-calcium-imaging/api/element_calcium_imaging/scan))
+
+| Table | Description |
+| --- | --- |
+| AcquisitionSoftware | Software used in the acquisiton of the imaging scans |
+| Channel | Recording Channel |
+| Scan | A set of imaging scans perfomed in a single session |
+| ScanLocation | Anatomical location of the region scanned |
+| ScanInfo | Metadata of the imaging scan |
+| ScanInfo.Field | Metadata of the fields imaged |
+| ScanInfo.ScanFile | Path of the scan file |
+
+### `imaging` schema ([API docs](https://datajoint.com/docs/elements/element-calcium-imaging/api/element_calcium_imaging/imaging))
+
+| Table | Description |
+| --- | --- |
+| ProcessingMethod | Available analysis suites that can be used in processing of the imaging scans |
+| ProcessingParamSet | All parameters required to process a calcium imaging scan |
+| CellCompartment | ?? |
```

Comment:
Please add descriptions where there are currently `??` and empty cells. This can be the same as the table description